backend/src/v5/models/history.js

```diff
@@ -0,0 +1,21 @@
+/**
+ *  Copyright (C) 2025 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* istanbul ignore file */
+const { v4Path } = require('../../interop');
```

Comment:
this file already exists in v5, but it's called revisions instead

https://github.com/3drepo/3drepo.io/blob/master/backend/src/v5/models/revisions.js

And I don't feel like we want to use anything inside v4 for this... why do you need it?

Reply:
It was to bring in the v4 createRevision method before you point out we have a new one - I've removed it!